Tests/libhostmgrTests/Model/PathsTests.swift

```diff
@@ -0,0 +1,58 @@
+import XCTest
+@testable import libhostmgr
+
+final class PathsTests: XCTestCase {
+
+    func testThatStorageRootIsCorrect() {
+        validate(path: Paths.storageRoot, resolvesTo: "/usr/local/var", forArchitecture: .x64)
+        validate(path: Paths.storageRoot, resolvesTo: "/opt/homebrew/var", forArchitecture: .arm64)
+    }
+
+    func testThatConfigurationRootIsCorrect() {
+        validate(path: Paths.configurationRoot, resolvesTo: "/usr/local/etc/hostmgr", forArchitecture: .x64)
+        validate(path: Paths.configurationRoot, resolvesTo: "/opt/homebrew/etc/hostmgr", forArchitecture: .arm64)
+    }
+
+    func testThatStateRootIsCorrect() {
+        validate(path: Paths.stateRoot, resolvesTo: "/usr/local/var/hostmgr/state", forArchitecture: .x64)
+        validate(path: Paths.stateRoot, resolvesTo: "/opt/homebrew/var/hostmgr/state", forArchitecture: .arm64)
+    }
+
+    func testThatVMStoragePathIsCorrect() {
+        validate(path: Paths.vmImageStorageDirectory, resolvesTo: "/usr/local/var/vm-images", forArchitecture: .x64)
+        validate(path: Paths.vmImageStorageDirectory, resolvesTo: "/opt/homebrew/vm-images", forArchitecture: .arm64)
+    }
+
+    func testThatGitMirrorStoragePathIsCorrect() {
+        let path = Paths.gitMirrorStorageDirectory
+        validate(path: path, resolvesTo: "/usr/local/var/git-mirrors", forArchitecture: .x64)
+        validate(path: path, resolvesTo: "/opt/homebrew/git-mirrors", forArchitecture: .arm64)
+    }
+
+    @available(macOS 13.0, *)
+    func testThatAuthorizedKeysFilePathIsCorrect() {
+        let resolvedPath = NSHomeDirectory() + "/.ssh/authorized_keys"
+        validate(path: Paths.authorizedKeysFilePath, resolvesTo: resolvedPath)
+    }
+
+    func testThatConfigurationFilePathIsCorrect() {
+        let path = Paths.configurationFilePath
+        validate(path: path, resolvesTo: "/usr/local/etc/hostmgr/config.json", forArchitecture: .x64)
+        validate(path: path, resolvesTo: "/opt/homebrew/etc/hostmgr/config.json", forArchitecture: .arm64)
+    }
+
+    private func validate(
+        path: URL,
+        resolvesTo sample: String,
```

Comment:
Nitpick. What do you think of calling this `expected`? As in, this is the expected value the path resolves to.

Reply:
I see what you're going for here for sure, but I feel like it reads a little weird – I tried `matches` which was a small improvement, then `matchesExpected` which is a bigger one, but then it felt a little wrong in context?

idk, I'll leave it for the moment – I agree with you that it's not _quite_ right but it's probably not worth overthinking at this point IMHO 😅